packages/atomic/src/components/atomic-result/atomic-result-cell-desktop.pcss

```diff
@@ -33,7 +33,7 @@
     atomic-result-section-badges {
       --font-size: var(--atomic-text-sm);
       font-size: var(--font-size);
-      --line-height: 1rem;
+      --line-height: calc(var(--font-size) * var(--atomic-line-height-ratio));
       line-height: var(--line-height);
```

Comment:
Perhaps we should extract these three lines into a PostCSS mixin, especially the line height variable definition.